nnvm/python/nnvm/frontend/keras.py

```diff
@@ -388,6 +388,38 @@ def _convert_reshape(insym, keras_layer, _):
     shape = (-1, ch) + keras_layer.target_shape[:-1]
     return _sym.reshape(insym, shape=shape)
 
+def _convert_lstm(insym, keras_layer, symtab):
+    _check_data_format(keras_layer)
+    if not isinstance(insym, list):
```

Comment:
Looks like insym is always list with the current tests.  Can you add another testcase which generates a non-list insym here?

Reply:
```
                if len(insym) == 1:
                    insym = insym[0]
```
if only one input is there, insym is not a list. so in this case, I'm making the state inputs and adding to the insym list.

Reply:
This can be added only by multiple lstm cells, i have added one testcase `test_LSTM_MultiLayer`